### PR TITLE
Generic Webhook enhancements

### PR DESCRIPTION
**Pull request by mohlcyber**
## Status
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
Added request header information to rawJSON output

## Must have
- [ ] Tests
- [x] Documentation

### CODE REVIEW

**Review by michal-dagan — COMMENTED** [review]
Hi @mohlcyber,
Thank you for your contribution!
Possibly, this will break compatibility with users already using the integration and extracting the rawJSON without accessing the `body` field, so I suggest adding the `headers` key to `raw_json` without the `body` key.
Please feel free to reach out to me with any questions - I'm available here or on slack :)
Thanks again

**Comment by mohlcyber** [comment]
Thanks @michal-dagan for the note. Made the corresponding changes.

**Comment by mohlcyber** [comment]
Demo recording has been published here: https://drive.google.com/file/d/1HusZHN1cJrFUqTkddkT1UsXh11jId29N/view?usp=sharing
If you have any questions please let me know.

File: Packs/GenericWebhook/ReleaseNotes/1_0_25.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#### Integrations
+
+##### Generic Webhook
+
+- Added the request header information to the output.
+- Updated the output to include a separate key for request headers inside raw_json field.
```
Comment (michal-dagan) [inline Packs/GenericWebhook/ReleaseNotes/1_0_25.md:6]:
```suggestion
- Updated the output to include a separate key for the request headers inside the rawJSON field.
```

**Comment by ShirleyDenkberg** [comment]
@michal-dagan Looks good.  Doc review completed.

File: Packs/GenericWebhook/Integrations/GenericWebhook/GenericWebhook.py [hunk 2]
```diff
@@ -56,7 +57,12 @@ async def handle_post(
         credentials: HTTPBasicCredentials = Depends(basic_auth),
         token: APIKey = Depends(token_auth)
 ):
+    header_name = None
+    request_headers = dict(request.headers)
+    secret_header = (header_name or 'Authorization').lower()
```
Comment (michal-dagan) [inline Packs/GenericWebhook/Integrations/GenericWebhook/GenericWebhook.py:62]: minor comment:
the header_name is only defined in line 71, so we need the secret_header to be defined also after
and you can delete the header_name definition in line 69, and revert line 74

Comment (mohlcyber) [reply Packs/GenericWebhook/Integrations/GenericWebhook/GenericWebhook.py:62]: yes makes sense. Just did another commit/push

**Review by michal-dagan — APPROVED** [review]

**Pull request merged**